src/type.h

```diff
@@ -672,7 +672,8 @@ class ArrayType : public SequentialType {
 
     int GetElementCount() const;
     const Type *GetElementType() const;
-
+    /* Returns true if array is unsized*/
```

Comment:
Again, would be give some definition of "unsized".

Reply:
Done